emoji_fzf/emoji_fzf.py

```diff
@@ -28,13 +29,34 @@ def cli():
     default=False,
     show_default=True,
 )
-def preview(prepend_emoji=False):
+@click.option(
+    "-c",
+    "--custom-aliases",
+    "custom_aliases_file",
+    help="Path to custom alias JSON file",
+    type=click.File("r"),
+    default=None,
+)
+def preview(prepend_emoji=False, custom_aliases_file=None):
```

Comment:
Maybe add this option to the `cli()` command group via https://click.palletsprojects.com/en/7.x/commands/#nested-handling-and-contexts
And update the `EMOJIS` dictionary there, so the aliases are available to the other commands too?

Reply:
Implemented in f6c52f7